optax/_src/alias.py

```diff
@@ -83,6 +83,7 @@ def adafactor(
     weight_decay_rate: Optional[float] = None,
     eps: float = 1e-30,
     factored: bool = True,
+    weight_decay_mask: Optional[Union[Any, Callable[[base.Params], Any]]] = None,
```

Comment:
I think `weight_decay_mask` is definitely a better name.
Should we update all `mask` in existing optimizers for weight decay to `weight_decay_mask`?

Reply:
I mean these lines @mtthss 
https://github.com/deepmind/optax/blob/3d666d92211dc0e2689c3acf82656d6286952733/optax/_src/alias.py#L354
https://github.com/deepmind/optax/blob/3d666d92211dc0e2689c3acf82656d6286952733/optax/_src/alias.py#L223